src/components/app-wrapper.tsx

```diff
@@ -86,38 +89,38 @@ const darkTheme = createTheme({
     mapboxStyle: 'mapbox://styles/mapbox/dark-v9',
 });
 
-const getMuiTheme = (theme) => {
+const getMuiTheme = (theme: unknown): Theme => {
```

Comment:
because it's coming from commons-ui, and its unknown JS data

Reply:
Then why do we do that ?
```
export function getLocalStorageTheme(): ReturnType<
    (typeof Storage.prototype)['getItem']
> {
    return localStorage.getItem(LOCAL_STORAGE_THEME_KEY) || DARK_THEME;
}
```
getItem returns string | null and you coalesce it with DARK_THEME, so we implicitly assume that DARK_THEME is a string. Let's assume it explicitly ! Typescript is here to help us understand better our code because it doesn't change anything at runtime anyway. So if we import strings from another lib where types are not defined, we have to define it ourselves so our code is more understandable and maintainable.